install/scripts/install-hipsycl.sh

```diff
@@ -15,6 +17,19 @@ git clone --recurse-submodules -b $HIPSYCL_REPO_BRANCH https://github.com/$HIPSY
 mkdir -p $BUILD_DIR/build
 cd $BUILD_DIR/build
 
+case $HIPSYCL_PKG_LLVM_REPO_BRANCH in
+	release/9.x)
+		LLVM_INCLUDE_PATH=$INSTALL_PREFIX/llvm/lib/clang/9.0.1/include
+		;;
+	release/10.x)
+		LLVM_INCLUDE_PATH=$INSTALL_PREFIX/llvm/lib/clang/10.0.1/include 
+		;;
+	*)
+		echo "LLVM version $HIPSYCL_PKG_LLVM_REPO_BRANCH is not supported"
+		exit -1
+		;;
```

Comment:
I wonder if we could improve this in the future such that we only have one set of variables for the clang version e.g. $MAJOR.$MINOR.$PATCH, such that it always includes `$INSTALL_PREFIX/llvm/lib/clang/$MAJOR.$MINOR.$PATH/include ` and pulls from `release/$MAJOR.x` or something similar. Or we just have a variable for the branch and a variable for the version that goes into the include path.

For now this case statement is good enough, but it's only going to get longer in the future ;)

Reply:
You're right, but If you agree I would change this right now, so we don't have to touch this code soon again

Reply:
It's fine if you want to do this change now, I just wanted to give you the option to get some closure with this PR for now since it has already been open for quite some time ;)